PlanetNode/GraphTypes/PlanetNodeMutation.cs

```diff
@@ -17,15 +17,17 @@ public PlanetNodeMutation(BlockChain<PolymorphicAction<PlanetAction>> blockChain
     {
         Field<TransactionMutation<PolymorphicAction<PlanetAction>>>(
             "transaction",
+            description: "Adds transaction to the pending list so that a next Block to be mined may contain given transaction.",
             resolve: context => new { }
         );
 
         Field<TransactionType<PolymorphicAction<PlanetAction>>>(
             "transferAsset",
+            description: "Mutates a new transaction about transfer and stage the transaction.",
```

Comment:
IMHO it's not a so-exactly description.  Here's some correction:

```suggestion
            description: "Transfers the given amount of PNG from the account of the specified " +
                "privateKeyHex to the specified recipient.  A made transaction is signed using " +
                "the privateKeyHex and added to the pending list (and eventually included in " +
                "one of the next blocks).",
```